rust-toolchain

```diff
@@ -1 +1 @@
-1.40.0
+1.41.0
```

Comment:
actually I believe this should be 1.41.1: https://blog.rust-lang.org/2020/02/27/Rust-1.41.1.html

Reply:
Gah, thanks. Updating.